src/observablemixin.js

```diff
@@ -674,6 +683,37 @@ function attachBindToListeners( observable, toBindings ) {
  * @param {*} oldValue The previous property value.
  */
 
+/**
+ * Fired when a property value is going to be changed but is not changed yet
+ * (before the {@link module:utils/observablemixin~ObservableMixin#change} event is fired).
```

Comment:
Incorrect link, unfortunately `module:utils/observablemixin~Observable#change` is incorrect as well.